src/server/grpcServer.go

```diff
@@ -101,14 +101,15 @@ func (s *workerServer) GetWorkerStatus(ctx context.Context, in *wpb.WorkerReques
 }
 
 func (s *workerServer) Convert(ctx context.Context, in *wpb.WorkerRequest) (*wpb.WorkerResponse, error) {
+
 	if in.GetPolicytype() == "network" {
 		log.Info().Msg("Convert network policy called")
 		networker.InitNetPolicyDiscoveryConfiguration()
-		networker.WriteNetworkPoliciesToFile("", "", []types.Service{})
+		return networker.ViewNetworkPolicies(), nil
```

Comment:
Please change it from ViewNetworkPolicies to GetNetworkPolicies

Reply:
There are already functions for GetNetworkPolicies and GetSystemPolicies which are called inside these functions.

Reply:
I will rename them to GetNetPolicy and GetSysPolicy, if that's fine too?